don't blindly overwrite $LD_LIBRARY_PATH
based on work by: Kenneth Hoste (HPC-UGent)
diff -ruN ncbi-vdb-3.0.0.orig/build/ld.linux.dlib.sh ncbi-vdb-3.0.0/build/ld.linux.dlib.sh
--- ncbi-vdb-3.0.0.orig/build/ld.linux.dlib.sh	2022-02-10 13:50:11.000000000 -0800
+++ ncbi-vdb-3.0.0/build/ld.linux.dlib.sh	2022-09-19 16:35:44.390187000 -0700
@@ -83,8 +83,7 @@
     done
 
     # update LD_LIBRARY_PATH
-    unset LD_LIBRARY_PATH
-    export LD_LIBRARY_PATH="$LDIRS:$XDIRS"
+    export LD_LIBRARY_PATH="$LDIRS:$XDIRS:$LD_LIBRARY_PATH"
 
     # tack on libraries, finding as we go
     for LIB in $LIBS
@@ -305,4 +304,4 @@
     echo "$MSG" >> $LOGFILE
 
     "$BUILD_DIR/scm-collect.sh" "$OBJS" "$SLIBS" | sort -u > "$TARG.md5"
-fi
\ No newline at end of file
+fi
diff -ruN ncbi-vdb-3.0.0.orig/build/ld.linux.exe.sh ncbi-vdb-3.0.0/build/ld.linux.exe.sh
--- ncbi-vdb-3.0.0.orig/build/ld.linux.exe.sh	2022-02-10 13:50:11.000000000 -0800
+++ ncbi-vdb-3.0.0/build/ld.linux.exe.sh	2022-09-19 16:36:32.114647000 -0700
@@ -91,8 +91,7 @@
     done
 
     # update LD_LIBRARY_PATH
-    unset LD_LIBRARY_PATH
-    export LD_LIBRARY_PATH="$LDIRS:$XDIRS"
+    export LD_LIBRARY_PATH="$LDIRS:$XDIRS:$LD_LIBRARY_PATH"
 
     # tack on libraries, finding as we go
     for LIB in $LIBS
